src/resources/js/tribe-common.js

```diff
@@ -0,0 +1,59 @@
+var tribe_auto_sysinfo = tribe_auto_sysinfo || {};
+
+tribe_auto_sysinfo.ajax = {
+	event: {}
+};
+
+(function ( $, my ) {
+	'use strict';
+
+	my.init = function () {
+		this.init_ajax();
+	};
+
+	/**
+	 * Initialize system info opt in
+	 */
+	my.init_ajax = function () {
+
+		this.$system_info_opt_in     = $( "#tribe_auto_sysinfo_opt_in" );
+		this.$system_info_opt_in_msg = $( ".tribe-sysinfo-optin-msg" );
+
+		this.$system_info_opt_in.change( function () {
+			if ( this.checked ) {
+				my.event.ajax( "generate" );
+			} else {
+				my.event.ajax( "remove" );
+			}
+
+		} );
+
+	};
+
+	my.event.ajax = function ( generate ) {
+
+		var request = {
+			"action": "tribe_toggle_sysinfo_optin",
+			"confirm": systeminfo.sysinfo_optin,
```

Comment:
Where is `systeminfo` defined? I'm guessing I'm just missing something obvious, but I don't see it localized.